source/read-write-concern/read-write-concern.rst

```diff
@@ -12,7 +12,7 @@ Read and Write Concern
 :Status: Approved
 :Type: Standards
 :Server Versions: 2.4+
-:Last Modified: December 18, 2017
+:Last Modified: January 29, 2019
 :Version: 1.5
```

Comment:
Since this is an informational change (not new functionality), I'd consider it a patch and would suggest bumping to 1.5.1.